app/src/main/res/values/strings.xml

```diff
@@ -241,7 +288,11 @@
 
     <!-- Attribution Strings -->
     <!--Common attribution string - https://www.usmint.gov/consumer/indexf8be.html?action=circCoinPolicy -->
-    <string name="attr_mint">United States coin images from the United States Mint</string>
+    <string name="attr_mint">Kennedy Half Dollar coin images from the United States Mint</string>
```

Comment:
This `attr_mint` attribution should stay generic, because it's the default used for most. If needed, we can split off a seprate `attr_kennedy` and use that for the Kennedy Half Dollars
```suggestion
    <string name="attr_mint">United States coin images from the United States Mint</string>
```